routes/zooRoute.js

```diff
@@ -0,0 +1,70 @@
+const router = require('express').Router();
+
+const knex = require('knex');
+const knexConfig = require('../knexfile');
+
+const db = knex(knexConfig.development);
+
+router.post('/', async (req, res) => {
```

Comment:
This is technically working in that it adds to the db, but it's returning the error message on my Postman as though it didn't work. Really weird! That happened before, and I assume it's not happening on your end. Did you get that figured out the last time it happened?